.github/workflows/reviewdog.yml

```diff
@@ -24,5 +24,5 @@ jobs:
           golangci_lint_flags: "-c .golangci.yml --allow-parallel-runners"
           reporter: "github-pr-review"
           tool_name: "Lint Errors"
-          level: "error"
```

Comment:
Changing the log level on CI lint will make it so that lint warnings cause failure correct? It's fine if we want that to be the standard--I just want to make sure I understand.

Reply:
Sorry, that was just for testing (and likely not relevant) - there is currently a panic caused in the Github action and I'm in the process of debugging this